travis/travis_install_nightly

```diff
@@ -38,6 +38,12 @@ if [ "${PHANTOMJS_VERSION}" != "OS" ]; then
     ln -s "${TRAVIS_BUILD_DIR}/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs" "${HOME}/maintainer-quality-tools/travis/phantomjs"
 fi
 
+# Update PyOpenSSL
+if [ "${PYOPENSSL_VERSION}" != "OS" ]; then
+    pip uninstall pyopenssl
+    pip install pyopenssl
```

Comment:
What about added to MQT/requirements.txt since that we are using -U to install them?